### PR TITLE
For #240: added UriBuilder wrapper and tests

### PR DESCRIPTION
**Pull request by paulodamaso**
For #240: 
- added FilteredUriBuilder wrapper
- added FilteredUriBuilderTest class with tests
- updated puzzle

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#242](https://github.com/amihaiemil/docker-java-api/issues/242) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#242](https://github.com/amihaiemil/docker-java-api/issues/242) is assigned to @sbespalov[/z](https://www.0crat.com/u/sbespalov), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/4b267323-7d18-4080-a176-91e3a54e75da); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @amihaiemil[/z](https://www.0crat.com/u/amihaiemil) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/main/java/com/amihaiemil/docker/FilteredUriBuilder.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import org.apache.http.client.utils.URIBuilder;
+
+import java.net.URI;
+import java.util.Map;
+
+/**
+ * {@link URIBuilder} with filtering.
+ * @author Paulo Lobo (pauloeduardolobo@gmail.com)
+ * @version $Id$
+ * @since 0.0.1
```
Comment (amihaiemil) [inline src/main/java/com/amihaiemil/docker/FilteredUriBuilder.java]: @paulodamaso don't forget the version here, it should be ``0.0.7``

**Comment by sbespalov** [comment]
@paulodamaso shouldn't coverage be fixed?
@amihaiemil also requested some changes regarding `FilteredUriBuilder` version.

**Comment by paulodamaso** [comment]
@sbespalov I think that adding a test which asserts that a method does not work is not a good thing, so I did not added test for these methods.

**Comment by sbespalov** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@rultor merge it

**Comment by 0crat** [comment]
Job `gh:amihaiemil/docker-java-api#242` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#242](https://github.com/amihaiemil/docker-java-api/issues/242) is now out of scope

**Pull request merged**